brigade.js

```diff
@@ -0,0 +1,42 @@
+const { events, Job } = require('brigadier');
+
+const coverage = (event, project) => {
+    console.log(`===> Building ${ project.repo.cloneURL } ${ event.commit }`);
+
+    const job = new Job('coverage', 'php:rc-alpine');
```

Comment:
Maybe can you start from the `composer` image directly ?

Reply:
:+1:  Like I did in the DockerFile: https://gist.github.com/soyuka/6aa9da5649907372279a88e4eb0555ef

Reply:
We should use a custom Docker image stored on the `coopTilleuls` account to save some CPU and speed up the builds.

Reply:
Done @dunglas. Can you please review?